zipkin-lens/src/components/App/App.test.js

```diff
@@ -11,54 +11,61 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-import { shallow } from 'enzyme';
```

Comment:
is enzyme still used directly? if not, do we need to change package.json?

Reply:
Yes Enzyme is still used in other components.